From ac5e98064a860055c03c8db4c148079ec25a0200 Mon Sep 17 00:00:00 2001
From: "Christian@p620" <christian.mardmoeller@renesas.com>
Date: Mon, 26 Sep 2022 17:52:27 +0200
Subject: [PATCH 55/92] drivers: net: lan865x: use 32-bit SPI words

All SPI transfer sizes are multiple of 4 bytes.

Using word size of 32 bits enables more efficient operation.
On Renesas VC4, this helps to prevent overruns at 10 MHz SPI clock.
---
 drivers/net/ethernet/microchip/lan865x/lan865x.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/microchip/lan865x/lan865x.c b/drivers/net/ethernet/microchip/lan865x/lan865x.c
index 073174a1e35e..14cebd54db74 100755
--- a/drivers/net/ethernet/microchip/lan865x/lan865x.c
+++ b/drivers/net/ethernet/microchip/lan865x/lan865x.c
@@ -874,6 +874,7 @@ static int lan865x_probe(struct spi_device *spi)
 		eth_hw_addr_random(netdev);
 
 	spi->rt = true;
+	spi->bits_per_word = 32;
 	spi_setup(spi);
 	pr_debug("spi->irq: %d\n", spi->irq);
 
-- 
2.30.2

